src/transformers/debug_utils.py

```diff
@@ -104,12 +107,19 @@ class DebugUnderflowOverflow:
     fast-forward right to that area.
 
 
+    Early stopping:
```

Comment:
It's missing something here, no?

Reply:
What do you feel is missing?

I'm just sort of trying to delineate independent sections and I'm not sure how to do it inside a docstring. Any suggestions?

The "Early stopping" is an independent feature to either of the mode 1 or mode 2. and I didn't want it to "stick" to the mode 2 section.

Suggestions?

Reply:
I didn't understand the sentence below was in the same section since indent was the same, but this works.

Reply:
Do you have any suggestions on how to create sub-sections inside the docstring?

Reply:
Not really, I only now how to do that in the rst.